astropy/units/tests/test_format.py

```diff
@@ -712,6 +712,35 @@ def test_vounit_scale_factor(unit, vounit, number, scale, voscale):
     assert x.to_string(format="vounit") == voscale + vounit
 
 
+@pytest.mark.parametrize(
+    "unit, vounit",
+    [("m s^-1", "m/s"), ("s^-1", "1/s"), ("100 s^-2", "100/s**2")],
+)
+@pytest.mark.parametrize(
+    "fraction",
+    [True, "inline"],
+)
```

Comment:
This should fit on a single line. Is the [trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) intentional?

Reply:
Not sure were it came from, as black had nothing to change here. And IMO trailing commas in this case are normal black format for multi-line parameter lists.